cirq/contrib/noise_models/noise_models.py

```diff
@@ -142,3 +145,121 @@ def noisy_moment(self, moment: 'cirq.Moment',
+    ):
+        """A depolarizing noise model with damped readout error.
+
+        All error modes are specified on a per-qubit basis. To omit a given
+        error mode from the noise model, leave its map blank when initializing
+        this object.
+
+        Args:
+            depol_prob_map: Map of depolarizing probabilities for each qubit.
+            bitflip_prob: Probability of a bit-flip during measurement.
+            decay_prob: Probability of T1 decay during measurement.
+                Bitflip noise is applied first, then amplitude decay.
+        """
+        for prob_map, desc in [(depol_prob_map, "depolarization prob"),
+                               (bitflip_prob_map, "readout error prob"),
+                               (decay_prob_map, "readout decay prob")]:
+            if prob_map:
+                for qubit, prob in prob_map.items():
+                    value.validate_probability(prob, f'{desc} of {qubit}')
+        self.depol_prob_map = depol_prob_map
+        self.bitflip_prob_map = bitflip_prob_map
+        self.decay_prob_map = decay_prob_map
+
+    def noisy_moment(self, moment: 'cirq.Moment',
+                     system_qubits: Sequence['cirq.Qid']):
+        moments = [moment]
+        if _homogeneous_moment_is_measurements(moment):
+            if self.decay_prob_map:
+                moments.insert(
+                    0,
+                    ops.Moment(
+                        ops.AmplitudeDampingChannel(self.decay_prob_map[q])(q)
+                        for q in system_qubits))
+            if self.bitflip_prob_map:
+                moments.insert(
+                    0,
+                    ops.Moment(
+                        ops.BitFlipChannel(self.bitflip_prob_map[q])(q)
+                        for q in system_qubits))
+            return moments
+        else:
+            if self.depol_prob_map:
+                gated_qubits = [
+                    q for q in system_qubits
+                    if moment.operates_on_single_qubit(q)
+                ]
+                moments.append(
+                    ops.Moment(
+                        ops.DepolarizingChannel(self.depol_prob_map[q])(q)
+                        for q in gated_qubits))
+            return moments
+
+
+def simple_noise_from_calibration_metrics(calibration: engine.Calibration,
+                                          depolNoise: bool = False,
+                                          dampingNoise: bool = False,
+                                          readoutDecayNoise: bool = False,
+                                          readoutErrorNoise: bool = False
+                                         ) -> devices.NoiseModel:
+    """Creates a reasonable PerQubitDepolarizingNoiseModel using the provided
+    calibration data. This object can be retrived from the engine by calling
+    'get_latest_calibration()' or 'get_calibration()' using the ID of the
+    target processor.
+    """
```

Comment:
How about adding Args and Returns sections to this docstring? I don't know what an engine.Calibration is. Looking at the code, it seems like you're making some implicit assumptions about the structure of `calibration` which should be made clear.